Session.js

```diff
@@ -733,7 +748,7 @@ Session.prototype = {
 			throw new Error('Arguments passed to executeAsync must be an array');
 		}
 
-		return this._post('execute_async', {
+		return this._post('execute/async', {
```

Comment:
`execute_async` is used in older versions of Selenium, so we should be careful before just dropping it.

Reply:
I think that I this should be another PR since I will need a test case to test this.